modules/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"devDependencies": {
-		"@liferay/npm-scripts": "37.0.0",
+		"@liferay/npm-scripts": "37.2.0",
```

Comment:
This version isn't released yet, or is it? How does this work?

(...looks in lockfile...)

Got it...

```
"@liferay/npm-scripts@37.2.0":
  version "37.2.0"
  resolved "http://localhost:4873/@liferay%2fnpm-scripts/-/npm-scripts-37.2.0.tgz#ad2db01cf073eed592e3262cd65038a8345ff50e"
```

Reply:
That's the "DO NOT MERGE" commit. I pushed it because I need it for local development, but doesn't belong to the PR. 

Sorry for the noise.